test/system/700-play.bats

```diff
@@ -981,3 +981,45 @@ _EOF
     run_podman pod rm -t 0 -f test_pod
     run_podman rmi -f userimage:latest $from_image
 }
+
+@test "podman play with automount volume" {
+    cat >$PODMAN_TMPDIR/Containerfile <<EOF
+FROM $IMAGE
+RUN sh -c 'mkdir /test1 /test2'
+RUN sh -c 'touch /test1/a /test1/b /test1/c'
+RUN sh -c 'touch /test2/asdf /test2/ejgre /test2/lteghe'
```

Comment:
Why the `sh -c`? All of those commands should work just fine with RUN. (Update: they do)